COUN-2477: DSFKit | DRA v4.14 adjustments

idannof-imperva opened the pull request:
COUN-2477: DSFKit | DRA v4.14 adjustments
- add official v4.14.0.10, make it default v4.14
- add official v4.13.0.20, make it default v4.13
- replace pre-release v4.13.0.10 with the official v4.13.0.10

idannof-imperva committed:
COUN-2477: DSFKit | DRA v4.14 adjustments
- bump default DRA version from v4.13 to v4.14

idannof-imperva committed:
COUN-2477: DSFKit | DRA v4.14 adjustments
- add official v4.14.0.10, make it default v4.14
- add official v4.13.0.20, make it default v4.13
- replace pre-release v4.13.0.10 with the official v4.13.0.10

lindanasredin commented on modules/aws/dra-admin/variables.tf:
```diff
@@ -27,8 +27,8 @@ variable "key_pair" {
-  default     = "4.13"
-  description = "The DRA version to install. Supported versions are 4.11.0.10.0.7 and up. Both long and short version formats are supported, for example, 4.11.0.10.0.7 or 4.11. The short format maps to the latest patch."
+  default     = "4.14"
+  description = "The DRA version to install. Supported versions are 4.11.0.10 and up. Both long and short version formats are supported, for example, 4.11.0.10 or 4.11. The short format maps to the latest patch."
```
There's a bug in the description, the modules accept only full version format, it's neither long nor short. So it should be:
"The DRA version to install. Supported versions are 4.11.0.10.0.7 and up. Full version format is supported, for example, 4.11.0.10.0.7."

idannof-imperva replied:
"the modules accept only full version format" -> just to make sure, the default value "4.14" that appears one line before the "description", will still work?

lindanasredin replied:
Yes, because the flow is:
1. User inputs 4.14 in example variable
2. In example code, when DRA module is instantiated, the example doesn't pass the version variable it received as it, it resolves it to the full version using the core/globals module.
3. The module accepts the full version

lindanasredin commented on modules/aws/dra-analytics/variables.tf:
```diff
@@ -82,8 +82,8 @@ variable "archiver_user" {
-  default     = "4.13"
-  description = "The DRA version to install. Supported versions are 4.11.0.10.0.7 and up. Both long and short version formats are supported, for example, 4.11.0.10.0.7 or 4.11. The short format maps to the latest patch."
+  default     = "4.14"
+  description = "The DRA version to install. Supported versions are 4.11.0.10 and up. Both long and short version formats are supported, for example, 4.11.0.10 or 4.11. The short format maps to the latest patch."
```
There's a bug in the description, the modules accept only full version format, it's neither long nor short. So it should be:
"The DRA version to install. Supported versions are 4.11.0.10.0.7 and up. Full version format is supported, for example, 4.11.0.10.0.7."

idannof-imperva committed:
COUN-2477: DSFKit | DRA v4.14 adjustments
- add official v4.14.0.10, make it default v4.14
- add official v4.13.0.20, make it default v4.13
- replace pre-release v4.13.0.10 with the official v4.13.0.10
